op-node/rollup/derive/l1_block_info.go

```diff
@@ -49,10 +54,17 @@ func (info *L1BlockInfo) MarshalBinary() ([]byte, error) {
 	copy(data[offset:offset+32], info.BlockHash.Bytes())
 	offset += 32
 	binary.BigEndian.PutUint64(data[offset+24:offset+32], info.SequenceNumber)
+	//offset += 32
+	//copy(data[offset+12:offset+32], info.BatcherAddr[:])
```

Comment:
In general I think we should be moving to a reader based API for marshalling/unmarshalling because these offset calculations are usually pretty tricky.

Reply:
Opened ENG-2965 to track this